### PR TITLE
Improve E2E tests for ES in OpenShift

### PR DESCRIPTION
**Pull request by iblancasa**
## Which problem is this PR solving?
- Before, all the E2E tests run in OpenShift were creating an autoprovisioned ES instance using the ES OpenShift Operator. This PR fixes that, creating external ES instances when required

## Short description of the changes
- Create a separate template for Jaeger instance creation autoprovisioning ES instances
- Other small fixes

### CODE REVIEW

**Review by frzifus — COMMENTED** [review]
As far as i understand its looking fine. Up to the others :)

File: tests/e2e/examples/render.sh [hunk 1]
```diff
@@ -12,7 +12,14 @@ render_smoke_test_example "$example_name" "02"
 
 start_test "examples-business-application-injected-sidecar"
 example_name="simplest"
-cat $EXAMPLES_DIR/business-application-injected-sidecar.yaml ./livenessProbe.yaml > ./00-install.yaml
+cp $EXAMPLES_DIR/business-application-injected-sidecar.yaml ./00-install.yaml
+$YQ e -i '.spec.template.spec.containers[0].livenessProbe.httpGet.path="/"' ./00-install.yaml
```
[the remark this answers is not in the document]
Comment (iblancasa) [reply tests/e2e/examples/render.sh:16]: Yes

**Comment by frzifus** [comment]
@kevinearls @rubenvp8510 could you please have a look too?

**Review by rubenvp8510 — APPROVED** [review]
LGTM

**Pull request merged**